girder/api/v1/system.py

```diff
@@ -194,37 +191,6 @@ def getVersion(self):
         version['serverStartDate'] = ModuleStartTime
         return version
 
-    @access.admin
-    @autoDescribeRoute(
-        Description('Set the list of enabled plugins for the system.')
-        .responseClass('Setting')
-        .notes('Must be a system administrator to call this.')
-        .jsonParam('plugins', 'JSON array of plugins to enable.', requireArray=True)
-        .errorResponse('Required dependencies do not exist.', 500)
-        .errorResponse('You are not a system administrator.', 403)
-    )
-    def enablePlugins(self, plugins):
-        # Determine what plugins have been disabled and remove their associated routes.
-        setting = Setting()
-        routeTable = setting.get(SettingKey.ROUTE_TABLE)
-        oldPlugins = setting.get(SettingKey.PLUGINS_ENABLED)
-        reservedRoutes = {GIRDER_ROUTE_ID, GIRDER_STATIC_ROUTE_ID}
-
-        routeTableChanged = False
-        removedRoutes = (
-            set(oldPlugins) - set(plugins) - reservedRoutes)
-
-        for route in removedRoutes:
```

Comment:
This logic is lost with this PR, but it only covered the case where a webroot was named exactly the same as the plugin, which is not enforced and doesn't cover plugins with multiple webroots.